cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt

```diff
@@ -0,0 +1,110 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+package de.fraunhofer.aisec.cpg.console
+
+import org.jetbrains.kotlinx.ki.shell.BaseCommand
+import org.jetbrains.kotlinx.ki.shell.Command
+import org.jetbrains.kotlinx.ki.shell.Plugin
+import org.jetbrains.kotlinx.ki.shell.Shell
+import org.jetbrains.kotlinx.ki.shell.configuration.ReplConfiguration
+
+class CompilationDatabase : Plugin {
+    inner class Load(conf: ReplConfiguration) : BaseCommand() {
+        override val name: String by conf.get(default = "translateCompilationDatabase")
+        override val short: String by conf.get(default = "trdb")
+        override val description: String =
+            "translates the source code files using the provided compilation database into the CPG"
+
+        override val params = "<path>"
+
+        override fun execute(line: String): Command.Result {
+            val p = line.indexOf(' ')
+            val path = line.substring(p + 1).trim()
+
+            return Command.Result.RunSnippets(
+                listOf(
+                    // basics
+                    "import de.fraunhofer.aisec.cpg.TranslationConfiguration",
+                    "import de.fraunhofer.aisec.cpg.TranslationManager",
+                    // go
+                    "import de.fraunhofer.aisec.cpg.frontends.golang.GoLanguageFrontend",
+                    // llvm
+                    "import de.fraunhofer.aisec.cpg.frontends.llvm.LLVMIRLanguageFrontend",
+                    // python
+                    "import de.fraunhofer.aisec.cpg.frontends.python.PythonLanguageFrontend",
+                    // all the graph nodes
+                    "import de.fraunhofer.aisec.cpg.graph.*",
+                    "import de.fraunhofer.aisec.cpg.graph.declarations.*",
+                    "import de.fraunhofer.aisec.cpg.graph.statements.*",
+                    "import de.fraunhofer.aisec.cpg.graph.statements.expressions.*",
+                    // helper builtins
+                    "import de.fraunhofer.aisec.cpg.graph.evaluate",
+                    "import de.fraunhofer.aisec.cpg.graph.all",
```

Comment:
I think this is now called `allChildren` since it was clashing with the `all` function of the new query API, correct @KuechA?

Reply:
Yes, that's right